styles/Microsoft/Acronyms.yml

```diff
@@ -0,0 +1,65 @@
+extends: conditional
+message: "'%s' has no definition."
+link: https://docs.microsoft.com/en-us/style-guide/acronyms
+
+level: suggestion
+ignorecase: false
+# Ensures that the existence of 'first' implies the existence of 'second'.
+first: '\b([A-Z]{3,5})\b'
+second: '(?:\b[A-Z][a-z]+ )+\(([A-Z]{3,5})\)'
+# ... with the exception of these:
+exceptions:
```

Comment:
A lot of Scaleway-specific acronyms are missing from this list. I suggest you sync it with the one used in the Go SDK: https://github.com/scaleway/scaleway-sdk-go/blob/master/strcase/goname.go#L102-L142

Reply:
I also started a list [here](https://github.com/scaleway/docs-content/pull/66/files?file-filters%5B%5D=.json&file-filters%5B%5D=.md&file-filters%5B%5D=.txt&file-filters%5B%5D=.yml#diff-747d026a1d56651b86d06324c386e8ec49655b5133665a6a2a5546472d06fc00)